Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3075,7 +3076,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
 #if PLATFORM(MAC)
     void substitutionsPanelIsShowing(CompletionHandler<void(bool)>&&);
-    void showCorrectionPanel(WebCore::AlternativeTextType panelType, const WebCore::FloatRect& boundingBoxOfReplacedString, String&& replacedString, String&& replacementString, Vector<String>&& alternativeReplacementStrings, WebCore::FrameIdentifier);
+    Awaitable<void> showCorrectionPanel(WebCore::AlternativeTextType panelType, WebCore::FloatRect boundingBoxOfReplacedString, String replacedString, String replacementString, Vector<String> alternativeReplacementStrings, WebCore::FrameIdentifier);
```

Comment:
why remove the r-values out of curiosity?

Reply:
references go out of scope when the first co_await makes the coroutine return control (unless the reference is copied or moved onto the call frame), but parameters passed by reference have the same lifetime as the coroutine.